.gitignore

```diff
@@ -1,4 +1,12 @@
-build/*
-dist/*
-genson.egg-info/*
+# See: github.com/github/gitignore/blob/master/Python.gitignore
```

Comment:
:+1: I usually keep this stuff in a global `.gitignore`, but not everyone does, so this is good practice. Thanks for doing the research and compiling this.